packages/react-devtools-shared/src/devtools/views/Components/InspectedElementContext.js

```diff
@@ -78,6 +72,11 @@ export function InspectedElementContextController({children}: Props) {
   const store = useContext(StoreContext);
   const {parseHookNames: parseHookNamesByDefault} = useContext(SettingsContext);
 
+  // parseHookNames has a lot of code.
+  // Embedding it into a build makes the build large.
+  // This function enables DevTools to make use of Suspense to lazily import() it only if the feature will be used.
+  const hookNamesModuleLoader = useContext(HookNamesModuleLoaderContext);
```

Comment:
just curious, why did you opt to make this one a new context?

Reply:
oh nvm, i just realized that the file we're importing in inline vs extensions is different